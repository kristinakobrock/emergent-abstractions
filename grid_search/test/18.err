  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:02, 21.50it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:02, 21.19it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 21.51it/s] 26%|#######################9                                                                    | 13/50 [00:00<00:01, 24.67it/s] 32%|#############################4                                                              | 16/50 [00:00<00:01, 23.53it/s] 38%|##################################9                                                         | 19/50 [00:00<00:01, 22.84it/s] 46%|##########################################3                                                 | 23/50 [00:00<00:01, 26.14it/s] 52%|###############################################8                                            | 26/50 [00:01<00:00, 25.99it/s] 58%|#####################################################3                                      | 29/50 [00:01<00:00, 26.97it/s] 64%|##########################################################8                                 | 32/50 [00:01<00:00, 27.63it/s] 70%|################################################################3                           | 35/50 [00:01<00:00, 24.76it/s] 78%|#######################################################################7                    | 39/50 [00:01<00:00, 26.26it/s] 84%|#############################################################################2              | 42/50 [00:01<00:00, 25.88it/s] 90%|##################################################################################8         | 45/50 [00:01<00:00, 26.80it/s] 98%|##########################################################################################1 | 49/50 [00:01<00:00, 27.73it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.44it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 24.67it/s] 54%|##################################################                                           | 7/13 [00:00<00:00, 28.04it/s] 77%|######################################################################7                     | 10/13 [00:00<00:00, 26.90it/s]100%|############################################################################################| 13/13 [00:00<00:00, 24.95it/s]100%|############################################################################################| 13/13 [00:00<00:00, 25.64it/s]
