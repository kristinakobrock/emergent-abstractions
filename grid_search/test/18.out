# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7001635432243347, "acc": 0.5775431990623474, "length": 3.5234222412109375, "mode": "train", "epoch": 1}
{"loss": 0.6968794465065002, "acc": 0.49667975306510925, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6696606874465942, "acc": 0.6252528429031372, "length": 3.3483049869537354, "mode": "train", "epoch": 2}
{"loss": 0.6940715312957764, "acc": 0.5130859017372131, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6648673415184021, "acc": 0.6200775504112244, "length": 3.187922716140747, "mode": "train", "epoch": 3}
{"loss": 0.6725662350654602, "acc": 0.6070312857627869, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6612281203269958, "acc": 0.6272056102752686, "length": 3.1058223247528076, "mode": "train", "epoch": 4}
{"loss": 0.6697593927383423, "acc": 0.6123046278953552, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6588150262832642, "acc": 0.6286313533782959, "length": 2.976738929748535, "mode": "train", "epoch": 5}
{"loss": 0.6661906242370605, "acc": 0.619140625, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6569480895996094, "acc": 0.6282288432121277, "length": 2.8134076595306396, "mode": "train", "epoch": 6}
{"loss": 0.6621405482292175, "acc": 0.6279296875, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6558207273483276, "acc": 0.6280561089515686, "length": 2.562318801879883, "mode": "train", "epoch": 7}
{"loss": 0.6622637510299683, "acc": 0.6267578601837158, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6524480581283569, "acc": 0.6286318898200989, "length": 2.438527822494507, "mode": "train", "epoch": 8}
{"loss": 0.6503162980079651, "acc": 0.6343750357627869, "length": 1.5390625, "mode": "test", "epoch": 8}
{"loss": 0.6408398747444153, "acc": 0.6355645060539246, "length": 2.720142126083374, "mode": "train", "epoch": 9}
{"loss": 0.6381436586380005, "acc": 0.6433379650115967, "length": 2.0, "mode": "test", "epoch": 9}
{"loss": 0.6264108419418335, "acc": 0.6531161665916443, "length": 2.9264822006225586, "mode": "train", "epoch": 10}
{"loss": 0.6297696828842163, "acc": 0.6462005972862244, "length": 2.96484375, "mode": "test", "epoch": 10}
{"loss": 0.6114975810050964, "acc": 0.6647953391075134, "length": 3.0782907009124756, "mode": "train", "epoch": 11}
{"loss": 0.6200693845748901, "acc": 0.6565139293670654, "length": 2.87890625, "mode": "test", "epoch": 11}
{"loss": 0.5970128178596497, "acc": 0.6795538663864136, "length": 3.2211811542510986, "mode": "train", "epoch": 12}
{"loss": 0.6180775165557861, "acc": 0.6627777814865112, "length": 2.91015625, "mode": "test", "epoch": 12}
{"loss": 0.5847249031066895, "acc": 0.6934794187545776, "length": 3.3110251426696777, "mode": "train", "epoch": 13}
{"loss": 0.6114380359649658, "acc": 0.6642287969589233, "length": 3.4765625, "mode": "test", "epoch": 13}
{"loss": 0.5728849172592163, "acc": 0.7039914131164551, "length": 3.4753670692443848, "mode": "train", "epoch": 14}
{"loss": 0.6008901000022888, "acc": 0.6722167134284973, "length": 3.28125, "mode": "test", "epoch": 14}
{"loss": 0.5581299662590027, "acc": 0.7142395973205566, "length": 3.470416307449341, "mode": "train", "epoch": 15}
{"loss": 0.5949126482009888, "acc": 0.6745697259902954, "length": 3.4921875, "mode": "test", "epoch": 15}
{"loss": 0.5438488721847534, "acc": 0.722776472568512, "length": 3.5427451133728027, "mode": "train", "epoch": 16}
{"loss": 0.5909793972969055, "acc": 0.6829620599746704, "length": 3.6640625, "mode": "test", "epoch": 16}
{"loss": 0.5312587022781372, "acc": 0.7333290576934814, "length": 3.61004900932312, "mode": "train", "epoch": 17}
{"loss": 0.585209846496582, "acc": 0.6895202398300171, "length": 3.6953125, "mode": "test", "epoch": 17}
{"loss": 0.5191754102706909, "acc": 0.7419394254684448, "length": 3.6599161624908447, "mode": "train", "epoch": 18}
{"loss": 0.5712480545043945, "acc": 0.70611572265625, "length": 3.65625, "mode": "test", "epoch": 18}
{"loss": 0.5104160904884338, "acc": 0.7484483122825623, "length": 3.70603084564209, "mode": "train", "epoch": 19}
{"loss": 0.5746492147445679, "acc": 0.714739978313446, "length": 3.69921875, "mode": "test", "epoch": 19}
{"loss": 0.49750712513923645, "acc": 0.7570661306381226, "length": 3.74314022064209, "mode": "train", "epoch": 20}
{"loss": 0.5436803102493286, "acc": 0.730262815952301, "length": 3.62890625, "mode": "test", "epoch": 20}
{"loss": 0.4870733618736267, "acc": 0.7662228941917419, "length": 3.7188076972961426, "mode": "train", "epoch": 21}
{"loss": 0.536287784576416, "acc": 0.7421616315841675, "length": 3.7421875, "mode": "test", "epoch": 21}
{"loss": 0.47427576780319214, "acc": 0.7762681841850281, "length": 3.78464412689209, "mode": "train", "epoch": 22}
{"loss": 0.52735435962677, "acc": 0.746838390827179, "length": 3.75, "mode": "test", "epoch": 22}
{"loss": 0.46430453658103943, "acc": 0.7813331484794617, "length": 3.794989585876465, "mode": "train", "epoch": 23}
{"loss": 0.5211634635925293, "acc": 0.7609664797782898, "length": 3.6796875, "mode": "test", "epoch": 23}
{"loss": 0.4480474293231964, "acc": 0.7952129244804382, "length": 3.766509771347046, "mode": "train", "epoch": 24}
{"loss": 0.5108389854431152, "acc": 0.7594757676124573, "length": 3.73828125, "mode": "test", "epoch": 24}
{"loss": 0.4348902106285095, "acc": 0.8057979941368103, "length": 3.817596197128296, "mode": "train", "epoch": 25}
{"loss": 0.4882814288139343, "acc": 0.7811645269393921, "length": 3.75, "mode": "test", "epoch": 25}
{"loss": 0.41708603501319885, "acc": 0.8178487420082092, "length": 3.8185670375823975, "mode": "train", "epoch": 26}
{"loss": 0.4775654077529907, "acc": 0.7910110354423523, "length": 3.6953125, "mode": "test", "epoch": 26}
{"loss": 0.4066588282585144, "acc": 0.8264098763465881, "length": 3.779184341430664, "mode": "train", "epoch": 27}
{"loss": 0.46184295415878296, "acc": 0.7958465814590454, "length": 3.79296875, "mode": "test", "epoch": 27}
{"loss": 0.3870887756347656, "acc": 0.8360636234283447, "length": 3.8490302562713623, "mode": "train", "epoch": 28}
{"loss": 0.4683650732040405, "acc": 0.7997101545333862, "length": 3.74609375, "mode": "test", "epoch": 28}
{"loss": 0.38241997361183167, "acc": 0.8400145769119263, "length": 3.809251546859741, "mode": "train", "epoch": 29}
{"loss": 0.45282551646232605, "acc": 0.8112580180168152, "length": 3.76171875, "mode": "test", "epoch": 29}
{"loss": 0.37341323494911194, "acc": 0.8436459898948669, "length": 3.8407163619995117, "mode": "train", "epoch": 30}
{"loss": 0.4453505277633667, "acc": 0.8159988522529602, "length": 3.76171875, "mode": "test", "epoch": 30}
{"loss": 0.36947304010391235, "acc": 0.8491955399513245, "length": 3.7858784198760986, "mode": "train", "epoch": 31}
{"loss": 0.4471893906593323, "acc": 0.8179551362991333, "length": 3.7421875, "mode": "test", "epoch": 31}
{"loss": 0.36029675602912903, "acc": 0.8529749512672424, "length": 3.827005386352539, "mode": "train", "epoch": 32}
{"loss": 0.43329355120658875, "acc": 0.8242583870887756, "length": 3.765625, "mode": "test", "epoch": 32}
{"loss": 0.35470929741859436, "acc": 0.8550732135772705, "length": 3.848820686340332, "mode": "train", "epoch": 33}
{"loss": 0.4324040412902832, "acc": 0.8287582397460938, "length": 3.77734375, "mode": "test", "epoch": 33}
{"loss": 0.35242217779159546, "acc": 0.8595864772796631, "length": 3.814049482345581, "mode": "train", "epoch": 34}
{"loss": 0.42572641372680664, "acc": 0.8284850120544434, "length": 3.75, "mode": "test", "epoch": 34}
{"loss": 0.3448275923728943, "acc": 0.8599435687065125, "length": 3.816347599029541, "mode": "train", "epoch": 35}
{"loss": 0.4249746799468994, "acc": 0.8339172601699829, "length": 3.7421875, "mode": "test", "epoch": 35}
{"loss": 0.34462785720825195, "acc": 0.8605272769927979, "length": 3.8032307624816895, "mode": "train", "epoch": 36}
{"loss": 0.4477335214614868, "acc": 0.8318436145782471, "length": 3.83203125, "mode": "test", "epoch": 36}
{"loss": 0.33978158235549927, "acc": 0.8605928421020508, "length": 3.846022129058838, "mode": "train", "epoch": 37}
{"loss": 0.42199909687042236, "acc": 0.8347257971763611, "length": 3.8203125, "mode": "test", "epoch": 37}
{"loss": 0.3373607099056244, "acc": 0.8649435043334961, "length": 3.8513379096984863, "mode": "train", "epoch": 38}
{"loss": 0.4170476198196411, "acc": 0.8380478024482727, "length": 3.74609375, "mode": "test", "epoch": 38}
{"loss": 0.33765554428100586, "acc": 0.8638220429420471, "length": 3.8004002571105957, "mode": "train", "epoch": 39}
{"loss": 0.4102010726928711, "acc": 0.8392806649208069, "length": 3.78515625, "mode": "test", "epoch": 39}
{"loss": 0.3298684060573578, "acc": 0.8649894595146179, "length": 3.826542615890503, "mode": "train", "epoch": 40}
{"loss": 0.41997000575065613, "acc": 0.83343505859375, "length": 3.765625, "mode": "test", "epoch": 40}
{"loss": 0.3242596685886383, "acc": 0.8683058619499207, "length": 3.826824188232422, "mode": "train", "epoch": 41}
{"loss": 0.41651028394699097, "acc": 0.8374481201171875, "length": 3.76953125, "mode": "test", "epoch": 41}
{"loss": 0.32451462745666504, "acc": 0.8696293234825134, "length": 3.8152124881744385, "mode": "train", "epoch": 42}
{"loss": 0.4158838093280792, "acc": 0.8378632068634033, "length": 3.80859375, "mode": "test", "epoch": 42}
{"loss": 0.3198924660682678, "acc": 0.8734145164489746, "length": 3.8385543823242188, "mode": "train", "epoch": 43}
{"loss": 0.42016762495040894, "acc": 0.8346923589706421, "length": 3.79296875, "mode": "test", "epoch": 43}
{"loss": 0.3196823298931122, "acc": 0.8718754053115845, "length": 3.812417984008789, "mode": "train", "epoch": 44}
{"loss": 0.41483253240585327, "acc": 0.8403152823448181, "length": 3.75, "mode": "test", "epoch": 44}
{"loss": 0.31461504101753235, "acc": 0.8742148876190186, "length": 3.8022749423980713, "mode": "train", "epoch": 45}
{"loss": 0.4019879102706909, "acc": 0.8436172604560852, "length": 3.796875, "mode": "test", "epoch": 45}
{"loss": 0.31734731793403625, "acc": 0.8713037371635437, "length": 3.8332905769348145, "mode": "train", "epoch": 46}
{"loss": 0.41481637954711914, "acc": 0.8387572765350342, "length": 3.7890625, "mode": "test", "epoch": 46}
{"loss": 0.3144274353981018, "acc": 0.8744353652000427, "length": 3.824026584625244, "mode": "train", "epoch": 47}
{"loss": 0.4017667770385742, "acc": 0.84516441822052, "length": 3.80859375, "mode": "test", "epoch": 47}
{"loss": 0.3114455044269562, "acc": 0.8761149048805237, "length": 3.839637279510498, "mode": "train", "epoch": 48}
{"loss": 0.4045805335044861, "acc": 0.8432083129882812, "length": 3.78125, "mode": "test", "epoch": 48}
{"loss": 0.31149861216545105, "acc": 0.8774107694625854, "length": 3.791713237762451, "mode": "train", "epoch": 49}
{"loss": 0.39392274618148804, "acc": 0.8465422987937927, "length": 3.76171875, "mode": "test", "epoch": 49}
{"loss": 0.306121826171875, "acc": 0.8771368265151978, "length": 3.837893486022949, "mode": "train", "epoch": 50}
{"loss": 0.4002609848976135, "acc": 0.842486560344696, "length": 3.83203125, "mode": "test", "epoch": 50}
{"loss": 0.3086490333080292, "acc": 0.8746904134750366, "length": 3.843158483505249, "mode": "train", "epoch": 51}
{"loss": 0.39946430921554565, "acc": 0.8445632457733154, "length": 3.78125, "mode": "test", "epoch": 51}
{"loss": 0.30671030282974243, "acc": 0.8776353597640991, "length": 3.787580728530884, "mode": "train", "epoch": 52}
{"loss": 0.39896467328071594, "acc": 0.8452560305595398, "length": 3.8046875, "mode": "test", "epoch": 52}
{"loss": 0.3043974041938782, "acc": 0.8782305121421814, "length": 3.84890079498291, "mode": "train", "epoch": 53}
{"loss": 0.409598171710968, "acc": 0.8442872166633606, "length": 3.8125, "mode": "test", "epoch": 53}
{"loss": 0.2997221350669861, "acc": 0.8793838024139404, "length": 3.838376045227051, "mode": "train", "epoch": 54}
{"loss": 0.39182913303375244, "acc": 0.8498993515968323, "length": 3.8125, "mode": "test", "epoch": 54}
{"loss": 0.295684278011322, "acc": 0.881405234336853, "length": 3.834731101989746, "mode": "train", "epoch": 55}
{"loss": 0.39919722080230713, "acc": 0.8441146612167358, "length": 3.83203125, "mode": "test", "epoch": 55}
{"loss": 0.29471614956855774, "acc": 0.8836679458618164, "length": 3.831763744354248, "mode": "train", "epoch": 56}
{"loss": 0.3975633382797241, "acc": 0.8458327054977417, "length": 3.80859375, "mode": "test", "epoch": 56}
{"loss": 0.2898624837398529, "acc": 0.884279727935791, "length": 3.8349404335021973, "mode": "train", "epoch": 57}
{"loss": 0.3970770835876465, "acc": 0.8473678231239319, "length": 3.8125, "mode": "test", "epoch": 57}
{"loss": 0.2858875095844269, "acc": 0.885918915271759, "length": 3.832267999649048, "mode": "train", "epoch": 58}
{"loss": 0.39628517627716064, "acc": 0.8496170043945312, "length": 3.82421875, "mode": "test", "epoch": 58}
{"loss": 0.2836901843547821, "acc": 0.8876073956489563, "length": 3.8512320518493652, "mode": "train", "epoch": 59}
{"loss": 0.4011603593826294, "acc": 0.8473526239395142, "length": 3.82421875, "mode": "test", "epoch": 59}
{"loss": 0.28419211506843567, "acc": 0.887071967124939, "length": 3.8541345596313477, "mode": "train", "epoch": 60}
{"loss": 0.39886200428009033, "acc": 0.8478851318359375, "length": 3.80859375, "mode": "test", "epoch": 60}
test accuracy: 0.7653161287307739
